### DISCUSSION TITLE
mspider/troubleshoot/hosted-mesh-errors/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1766] [Announcements]
# mspider/troubleshoot/hosted-mesh-errors/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/troubleshoot/hosted-mesh-errors/

**Comment by git-jxj** [comment]
登录控制面集群执行以下命令获取这个地址的命令，命令中的istiod-ywistio-hosted-lb是动态的，需要替换
